### PR TITLE
Staff: change exception handling and input parsing

### PR DESCRIPTION
**Pull request by Lim-Shi-Han**

### CODE REVIEW

**Review by BotBw — DISMISSED** [review]
lgtm

File: src/main/java/seedu/duke/controllers/StaffController.java [hunk 1]
```diff
@@ -15,57 +17,53 @@
...
-            return true;
-        default:
-            System.out.println("Unknown choice!");
-            break;
+    protected boolean optionSwitcher(int choice) throws OperationTerminationException {
+        try {
+            switch (choice) {
+            case 1:
+                Staff staff = findStaff();
+                System.out.println(staff);
```
Comment (darylhjd) [inline src/main/java/seedu/duke/controllers/StaffController.java:33]: i would just put this into another method

File: src/main/java/seedu/duke/controllers/StaffController.java [hunk 2]
```diff
@@ -15,57 +17,53 @@
...
+                break;
+            case 0:
+                System.out.println("Exiting Staff Menu...");
+                return true;
+            default:
+                System.out.println("Unknown choice!");
+                break;
+            }
+        } catch (IllegalArgumentException e) {
+            System.out.println(e.getMessage());
```
Comment (darylhjd) [inline src/main/java/seedu/duke/controllers/StaffController.java:46]: where is this exception thrown? (is it needed)

**Review by darylhjd — CHANGES_REQUESTED** [review]

**Comment by BotBw** [comment]
should we deal with invalid input by throwing an exception (like IllegalArgumentException) at entities level and catch that exception at child controller level? (I print the message saying that the input is not valid, then return false in optionSwitcher to let user select the option and input again)

**Pull request merged**